src/nvim/event/loop.c

```diff
@@ -0,0 +1,43 @@
+#include <stdint.h>
+
+#include <uv.h>
+
+#include "nvim/event/loop.h"
+
+#ifdef INCLUDE_GENERATED_DECLARATIONS
+# include "event/loop.c.generated.h"
+#endif
+
+
+void loop_init(Loop *loop, void *data)
+{
+  uv_loop_init(&loop->uv);
+  loop->uv.data = loop;
+}
+
+void loop_run(Loop *loop)
+{
+  uv_run(&loop->uv, UV_RUN_DEFAULT);
```

Comment:
Not intentionally, for now this is just a stub module that I plan to finish later